src/api/__init__.py

```diff
@@ -1 +1,2 @@
-from .app import create_app
\ No newline at end of file
+# For api to be treated as a package, it must contain an __init__.py file 
+# so that blueprint imports can be done in the api.py file.
```

Comment:
You shouldn't need `__init.py__` for importing from modules within the same folder. May be the dot in `.pros` is the culprit. Try importing directly by doing `from pros import pros`. It should work. If so, you can drop the __init.py__ for now.

Reply:
I think this depends on where you are running the API from.